sdk/webpubsub/azure-messaging-webpubsubservice/tests/test_reverse_proxy_async.py

```diff
@@ -25,7 +25,7 @@ def _callback(pipeline_request):
         wps_endpoint = "https://wps.contoso.com/"
         apim_endpoint = "https://apim.contoso.com/"
         credential = AzureKeyCredential("AzureKeyCredential")
-        request = build_send_to_all_request("Hub", content='test_webpubsub_send_request', content_type='text/plain')
+        request = build_web_pub_sub_service_send_to_all_request("Hub", content='test_webpubsub_send_request', content_type='text/plain')
```

Comment:
Why is this test using the send_request method? If that is what we expect customers to use, why are we adding client level method for this?

Reply:
the send_to_all method that the customers use are tested in the other file I believe (test_smoke_py)

Reply:
Ok - I generally prefer testing through the public API surface area.